app/containers/FileModal.js

```diff
@@ -75,19 +98,31 @@ const ModalContent = React.memo(({
 		const uri = formatAttachmentUrl(attachment.video_url, user.id, user.token, baseUrl);
 		return (
 			<SafeAreaView style={styles.safeArea}>
```

Comment:
We can remove this `SafeAreaView`.
It's going to be like this:
<img src="https://user-images.githubusercontent.com/804994/63293230-f3f3b580-c29d-11e9-92bf-b9d8870e42d2.png" width="350" />